Controllers/v1/PetsController.cs

```diff
@@ -0,0 +1,140 @@
+using bootcamp_api.Exceptions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using AutoMapper;
+using bootcamp_api.Services;
+using Microsoft.Extensions.Hosting;
+using Dto;
+using System;
+
+namespace bootcamp_api.Controllers
+{
+
+    /// <summary>
+    /// Handles incoming HTTP requests for pets
+    /// </summary>
+    [ApiController]
+    [ApiVersion("1.0")]
+    [Produces("application/json")]
+    [Consumes("application/json")]
+    [Route("api/v{version:apiVersion}/[controller]")]
+    public class PetsController : ControllerBase
+    {
+
+        private readonly IPetService _petService;
+        private readonly IMapper _mapper;
+
+        /// <summary>
+        /// PetsController constructor
+        /// </summary>
+        public PetsController(IPetService petService, IMapper mapper)
+        {
+            _petService = petService;
+            _mapper = mapper;
+        }
+
+        /// <summary>
+        /// Returns all pets
+        /// </summary>
+        [HttpGet]
+        [ProducesResponseType(typeof(Pet[]), StatusCodes.Status200OK)]
+        public IActionResult GetAll()
```

Comment:
Your APi should provide features for your UI.  I dont think you have any pages that show all pets.  I think this would be better as a search function instead of returning everything.

Reply:
I modified GetAll to account for a user's ID and only return summary details as a PetListItem.